packages/gestalt-design-tokens/README.md

```diff
@@ -0,0 +1,181 @@
+# Basic Style Dictionary
+
+This example code is bare-bones to show you what this framework can do. If you have the style-dictionary module installed globally, you can `cd` into this directory and run:
+```bash
+style-dictionary build
```

Comment:
I'd add a little more guidance on how to get it running
command to install globally >> cd <path> to get there >> style-dictionary build